lib/github-comment.js

```diff
@@ -1,16 +1,55 @@
 'use strict'
 
 const githubClient = require('./github-client')
+const GQL = require('./github-graphql-client')
+
+const getPRComments = `query getPRComments($owner: String!, $repo: String!, $number: Int!, $cursor: String){
+  repository(owner: $owner, name:  $repo) {
+    pullRequest(number: $number) {
+      comments(first: 20, after:$cursor) {
+        nodes {
+          id
+          body
+          viewerDidAuthor
+        }
+        pageInfo {
+          endCursor
+          hasNextPage
+        }
+      }
+      labels(first: 15) {
+        nodes {
+          name
+        }
+      }
```

Comment:
Planning to use these labels for something?

Reply:
If we want to recognize a `bot-out` label, for complete opt-out.